db/migrate/20160708235959_add_web_hook_event.rb

```diff
@@ -0,0 +1,18 @@
+class AddWebHookEvent < ActiveRecord::Migration
+  def change
+    enable_extension 'uuid-ossp' unless extension_enabled?('uuid-ossp')
+
+    create_table :web_hook_events, id: :uuid, default: 'uuid_generate_v1mc()' do |t|
```

Comment:
hmm we have alot of posts being saved in parallel as well. Don't think we need a uuid in this case. I've never used this so I might be missing something important :stuck_out_tongue: